README.md

```diff
@@ -17,6 +17,20 @@ If you want to run the application locally in the production mode, run `mvn spri
 
 To run Integration Tests, execute `mvn verify -Pintegration-tests`.
 
+### Live Reload (optional)
+
+When you change your Java code and recompile it, the application will be automatically reloaded without need to restart it.
```

Comment:
Already good as-is, but a suggestions to explain the main idea: 

With live reload you can see results of your code changes immediately. When you edit your Java code and recompile it, the application changes will be automatically reloaded and browser is refreshed.

Reply:
Done.